<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hypermix2</class>
 <widget class="QMainWindow" name="Hypermix2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hypermix2</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>11</y>
      <width>771</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="VL">
     <item>
      <widget class="QGraphicsView" name="GV"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="HL">
       <item>
        <widget class="QPushButton" name="pushLSU">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>LSU</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushNCLSU">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>NCLSU</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushISRA">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>ISRA</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushFCLSU">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>FCLSU</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushCLEAR">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>CLEAR</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushCLEARALL">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>CLEAR ALL</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
